application/routes/vnc.py

```
@@ -19,15 +19,16 @@
 import json
-import logging
+
 
 from flask import request, abort, stream_with_context
 
 from .common import create_connection
 from .. import api, app, profiles
 from ..codes import ICtrlStep, ICtrlError, ConnectionType
 from ..utils import int_to_bytes
-
+import application
```

Ruff

`application` imported but unused

Remove unused import: `application`

(F401)